In [15]:
import matplotlib.pylab as plt
import numpy as np
import pandas as pd
%matplotlib inline

In [16]:
data = pd.read_csv("https://covid19.manaus.am.gov.br/wp-content/uploads/Manaus.csv",sep=';')

/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py:2718: DtypeWarning: Columns (0,6,7,8,9,10,11,12,13,14,15,16,17,18,22,23,26,27,28,29,30,31) have mixed types.Specify dtype option on import or set low_memory=False.
  interactivity=interactivity, compiler=compiler, result=result)


In [17]:
data.head()

_dt_evolucao   _dt_notificacao  ... _distrito        _taxa
0  25/01/2020 00:00  03/01/2020 00:00  ...     OESTE  0,243042897
1  03/01/2020 00:00  03/01/2020 00:00  ...     OESTE  0,684931507
2  05/01/2020 00:00  03/01/2020 00:00  ...     LESTE  0,073137374
3  16/01/2020 00:00  08/01/2020 00:00  ...       SUL  0,198306463
4  11/01/2020 00:00  08/01/2020 00:00  ...     NORTE  0,117699678

[5 rows x 39 columns]

In [18]:
data.tail()

_dt_evolucao   _dt_notificacao  ... _distrito        _taxa
243979          NaN  20/04/2020 00:00  ...       SUL   0,99960016
243980          NaN  20/04/2020 00:00  ...       SUL   0,99960016
243981          NaN  20/04/2020 00:00  ...     NORTE  0,068152389
243982          NaN  22/04/2020 00:00  ...       SUL  0,198306463
243983          NaN  22/04/2020 00:00  ...     OESTE  1,648804617

[5 rows x 39 columns]

1. Quantos atributos descrevem cada exemplo? Quais são eles?

In [19]:
data.shape

(243984, 39)

**Resposta**: Cada exemplo possui 39 atributos, sendo eles informados na célula a seguir:

In [20]:
data.columns

Index(['_dt_evolucao', '_dt_notificacao', '_dt_sintomas', '_classificacao',
       '_conclusao', '_origem', '_comorb_cardio', '_comorb_cromossomica',
       '_comorb_diabetes', '_comorb_hemato', '_comorb_hepatica',
       '_comorb_imuno', '_comorb_neurologica', '_comorb_obessidade',
       '_comorb_renal', '_comorb_respiratoria', '_criterio', '_etnia',
       '_evolução', '_idade', '_faixa etária', '_gestante', '_profiss_saude',
       '_raca', '_se_notificacao', '_sexo', '_sintoma_dispneia',
       '_sintoma_febre', '_sintoma_garganta', '_sintoma_tosse',
       '_sintoma_outros', '_srag', '_teste_anticorpo', '_teste_antigeno',
       '_teste_pcr', '_bairro', '_bairro_mapa', '_distrito', '_taxa'],
      dtype='object')

2. Quantos casos confirmados há em Manaus, cumulativamente?

In [21]:
data['_classificacao'].value_counts()

Descartado    126563
Confirmado     80420
Em análise     37001
Name: _classificacao, dtype: int64

**Resposta:** No dataset constam 80420 casos confirmados, 126563 casos descartados e 37001 casos em análise

3. A qual período de tempo a base de dados se refere, isto é, qual o registro mais antigo e qual o mais recente? Leve em conta a data de notificação.